tari-project/universe
# fix: reduce requests to tari suite

## #891 
### [ Content ]
- Added `release_cache_id` property for each binary
- Added mechanism to cache releases versions for the time of downloading and setting up binaries
- Added cache cleanup mechanism
------------
### [ Testing ]
- Remove all binaries
- force `should_check_for_update` flag to true ( main.rs::577 )
- Start app and check logs there should be no more then 1 download of releases versions for tari-suite ( mm_proxy, minotari_node, wallet )
